.github/workflows/ci.yml

```diff
@@ -72,16 +72,16 @@ jobs:
         with:
           source-folder: code
 
-      - name: Send mail
-        if: ${{ failure() }}
-        uses: dawidd6/action-send-mail@v3
-        with:
-          server_address: smtp.gmail.com
-          server_port: 465
-          username: ${{ secrets.MAIL_USERNAME }}
-          password: ${{ secrets.MAIL_PASSWORD }}
-          subject: ${{ github.job }} job of ${{ github.repository }} has ${{ job.status }}
-          to: ${{ secrets.CI_INTERNAL_EMAIL }}
-          from: ${{ secrets.MAIL_USERNAME }}
-          secure: true
-          body: ${{ github.job }} job in workflow ${{ github.workflow }} of ${{ github.repository }} has ${{ job.status }}
+      # - name: Send mail
+      #   if: ${{ failure() }}
+      #   uses: dawidd6/action-send-mail@v3
+      #   with:
+      #     server_address: smtp.gmail.com
+      #     server_port: 465
+      #     username: ${{ secrets.MAIL_USERNAME }}
+      #     password: ${{ secrets.MAIL_PASSWORD }}
+      #     subject: ${{ github.job }} job of ${{ github.repository }} has ${{ job.status }}
+      #     to: ${{ secrets.CI_INTERNAL_EMAIL }}
+      #     from: ${{ secrets.MAIL_USERNAME }}
+      #     secure: true
+      #     body: ${{ github.job }} job in workflow ${{ github.workflow }} of ${{ github.repository }} has ${{ job.status }}
```

Comment:
Rather than comment this whole thing out can we just change the if condition?

`if: ${{ failure() && runner.os == 'Linux' }}  # remove Linux check when Windows and Mac are working again`

Reply:
Good call. updated